doc/ref_arch/kubernetes/chapters/appendix-a.md

```diff
@@ -1,21 +1,25 @@
 [<< Back](../../kubernetes)
-# Appendix A - Guidance for VNF-to-CNF Transition
+# Appendix A - Guidelines
 <p align="right"><img src="../figures/bogo_ifo.png" alt="scope" title="Scope" width="35%"/></p>
 
 ## Table of Contents
-* [A.1 TBC](#A.1)
-* [A.2 TBC](#A.2)
-* [A.3 TBC](#A.3)
-* [A.4 TBC](#A.3)
+* [A.1 Guidance for Vendors](#A.1)
+* [A.2 RA Transitional Generations](#A.2)
+* [A.3 Guidance for VNF-to-CNF Transition](#A.3)
+* [A.4 TBC](#A.4)
 
 <a name="A.1"></a>
-## A.1 TBC
+## A.1 Guidance for Vendors
 
 <a name="A.2"></a>
-## A.2 TBC
+## A.2 RA Transitional Generations
+
+CNF Generation definitions are defined in [RM Appendix](../../../ref_modle/chapters/appendix-a#A.4)
```

Comment:
Typo: "../../../ref_modle/chapters/appendix-a#A.4" (also in other places) -- should be "../../../ref_model/chapters/appendix-a#A.4"

Reply:
should be fixed now